code/modules/mob/living/simple_animal/guardian/types/dextrous.dm

```diff
@@ -1,102 +1,33 @@
-		msg += "[desc]\n"
-
-		for(var/obj/item/I in held_items)
-			if(!(I.flags & ABSTRACT))
-				if(I.blood_DNA)
-					msg += "<span class='warning'>It has \icon[I] [I.gender==PLURAL?"some":"a"] blood-stained [I.name] in its [get_held_index_name(get_held_index_of_item(I))]!</span>\n"
-				else
-					msg += "It has \icon[I] \a [I] in its [get_held_index_name(get_held_index_of_item(I))].\n"
-
-		if(internal_storage && !(internal_storage.flags&ABSTRACT))
-			if(internal_storage.blood_DNA)
-				msg += "<span class='warning'>It is holding \icon[internal_storage] [internal_storage.gender==PLURAL?"some":"a"] blood-stained [internal_storage.name] in its internal storage!</span>\n"
-			else
-				msg += "It is holding \icon[internal_storage] \a [internal_storage] in its internal storage.\n"
-		msg += "*---------*</span>"
-		to_chat(user, msg)
-	else
-		..()
-
-/mob/living/simple_animal/hostile/guardian/dextrous/Recall(forced)
-	if(!summoner || loc == summoner || (cooldown > world.time && !forced))
+	/obj/machinery/camera)
+	guardian.melee_damage_lower += 5
+	guardian.melee_damage_upper += 5
+	for(var/i in guardian.damage_coeff)
+		guardian.damage_coeff[i] -= 0.15
+
+/datum/guardian_abilities/dextrous/recall_act(forced)
+	if(!user || guardian.loc == user || (cooldown > world.time && !forced) && guardian.dextrous)
 		return FALSE
-	drop_all_held_items()
-	return ..() //lose items, then return
-
-/mob/living/simple_animal/hostile/guardian/dextrous/snapback()
-	if(summoner && !(get_dist(get_turf(summoner),get_turf(src)) <= range))
-		drop_all_held_items()
-		..() //lose items, then return
-
-//SLOT HANDLING BULLSHIT FOR INTERNAL STORAGE
-/mob/living/simple_animal/hostile/guardian/dextrous/doUnEquip(obj/item/I, force)
-	if(..())
-		update_inv_hands()
-		if(I == internal_storage)
-			internal_storage = null
-			update_inv_internal_storage()
-		return 1
-	return 0
-
-/mob/living/simple_animal/hostile/guardian/dextrous/can_equip(obj/item/I, slot)
-	switch(slot)
-		if(slot_generic_dextrous_storage)
-			if(internal_storage)
-				return 0
-			return 1
-	..()
-
-/mob/living/simple_animal/hostile/guardian/dextrous/equip_to_slot(obj/item/I, slot)
-	if(!..())
-		return
-
-	switch(slot)
-		if(slot_generic_dextrous_storage)
-			internal_storage = I
-			update_inv_internal_storage()
-		else
-			to_chat(src, "<span class='danger'>You are trying to equip this item to an unsupported inventory slot. Report this to a coder!</span>")
-
-/mob/living/simple_animal/hostile/guardian/dextrous/getBackSlot()
-	return slot_generic_dextrous_storage
-
-/mob/living/simple_animal/hostile/guardian/dextrous/getBeltSlot()
-	return slot_generic_dextrous_storage
-
-/mob/living/simple_animal/hostile/guardian/dextrous/proc/update_inv_internal_storage()
-	if(internal_storage && client && hud_used && hud_used.hud_shown)
-		internal_storage.screen_loc = ui_id
-		client.screen += internal_storage
+	guardian.drop_all_held_items()
+	return TRUE //lose items, then return
```

Comment:
yes but the guardians can still just pick up the thing and use it

whatever ill edit the snapback act.